devel/protobuf2-cpp/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+PortGroup  github 1.0
+
+name            protobuf2-cpp
+version         2.6.1
+revision        1
+categories      devel
+maintainers     nomaintainer
+license         BSD
+description     Encode data in an efficient yet extensible format.
+
+long_description \
+                Google Protocol Buffers are a flexible, efficient, \
+                automated mechanism for serializing structured data -- \
+                think XML, but smaller, faster, and simpler.  You \
+                define how you want your data to be structured once, \
+                then you can use special generated source code to \
+                easily write and read your structured data to and from \
+                a variety of data streams and using a variety of \
+                languages.  You can even update your data structure \
+                without breaking deployed programs that are compiled \
+                against the "old" format.  You specify how you want \
+                the information you're serializing to be structured by \
+                defining protocol buffer message types in .proto \
+                files.  Each protocol buffer message is a small \
+                logical record of information, containing a series of \
+                name-value pairs.
+
+github.setup    google protobuf ${version} v
+github.tarball_from releases
+distname        protobuf-${version}
+dist_subdir     protobuf2-cpp
```

Comment:
For the various protobuf* ports, `dist_subdir` should be set to `protobuf-cpp`. I can elaborate further if you'd like, though.